uktrade/data-hub-frontend
# Fix/typeahead stories

## Description of change

Fixes the storybook for the typeahead component with a pre-selected option and adds some visual regression tests. I also fixed a warning that said 'page' had the wrong type.

## Test instructions

* Go to the storybook page for typeahead -> single select -> Options - pre-selected option
* 'Vanilla - Vanilla' should appear as the pre-selected option
* Selecting another option should change the selected option to that value

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
